app/views/content/non-uk-teachers/why-train-to-teach-in-england-video-transcript.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Why train to teach in England video transcript
+description: Full transcript for the 'Why train to teach in England' video.
+
+---
+
+## Teach in England
+### Why train to teach in England?
```

Comment:
@Sarah-DfE This isn't quite right
![image](https://github.com/user-attachments/assets/8a94d3f8-c706-4f8e-a6b5-8b8aca1702c7)

I think you just need to add a line in between - here's an example from the choosing a course page:
![image](https://github.com/user-attachments/assets/f286006a-af28-40a2-890b-fe886c2353c9)